views/movies/new-movie.hbs

```diff
@@ -0,0 +1,20 @@
+<h2>Add a new Movie</h2>
+<form action="/movies/create" method="post">
+
+  <label for="title">Title</label>
+  <input type="text" name="title" id="title" />
+
+  <label for="genre">Genre</label>
+  <input type="text" name="genre" id="genre" />
+
+  <label for="plot">Plot</label>
+  <input type="text" name="plot" id="plot" />
+
+  <label for="cast">Cast</label>
+
+<select name="cast" id="cast">
+    <option value="">Cast</option>
```

Comment:
So now, you can get all the celebrities you have on the select element by doing:

```
<select name="cast" id="cast">
    {{#each celebs}}
    <option value="{{_id}}">{{name}}</option>
    {{/each}}
</select>
```